Assets/TilemapHeightTest/Scripts/Player.cs

```diff
@@ -47,7 +47,9 @@ private void Update()
 
 			_body.velocity = new Vector2(horiz, vert) * MoveSpeed;
 
-			TileHeightManager.Instance.ReportPosition(transform.position, _sprite.sprite.bounds);
+			foreach (TileHeightManager Instance in TileHeightManager.Instances) {
```

Comment:
Nit: `Instance` should be `instance`. Prefer `{` in its own line. Prefer using `var` instead of `TileHeightManager`.